Doc/library/sndhdr.rst

```diff
@@ -49,3 +49,51 @@ be the sample size in bits or ``'A'`` for A-LAW or ``'U'`` for u-LAW.
    .. versionchanged:: 3.5
       Result changed from a tuple to a namedtuple.
 
+The following sound header types are recognized, as listed below with the return value
+from :func:`whathdr`: and :func:`what`:
+
++------------+------------------------------------+
+| Value      | Sound header format                |
++============+====================================+
+| ``'aifc'`` | Compressed Audio Interchange Files |
++------------+------------------------------------+
+| ``'aiff'`` | Audio Interchange Files            |
```

Comment:
Nice table!  Details: unnecessary capitalization for `files`; I think the table could be right at the beginning of the doc around line 31, and then you could remove the list of all values on lines 24-25.

Reply:
Am not a native english speaker but my impression is these are abbreviations hence capitalization, no? see, e.g https://en.wikipedia.org/wiki/Audio_Interchange_File_Format.

Reply:
Yes, as the title of a format I think this is fine to capitalise. But let's use the same names:

* Compressed Audio Interchange File Format
* Audio Interchange File Format

And then consistently for the others, if possible. This was the best I found in some quick searches and may need double checking, some of these are really old!

* Au
* HCOM
* Sounder
* SndTool
* Creative Voice
* Waveform Audio File Format
* 8-Bit Sampled Voice
* Signed Byte Audio Data
* Unsigned Byte Audio File
* uLAW Audio File


I'm not too sure about `sb`, ub` and `ul`: but more importantly looks like they were removed in 1999 and Python 2.0! 

https://github.com/python/cpython/commit/f4bf044999ad48684b8d70c229756a1da8a7613b

So let's ditch those and belatedly remove them from `SndHeaders.filetype.__doc__` and anywhere else :)